subgraph/src/datasources/FortaStaking.ts

```diff
@@ -341,6 +341,15 @@ export function handleWithdrawalExecuted(event: WithdrawalExecuted): void {
   withdrawalExecutedEvent.subject = formatSubjectId(event.params.subject, event.params.subjectType);
   withdrawalExecutedEvent.amount = withdrawalInitiatedEvent.amount;
   withdrawalExecutedEvent.save();
+
+  if (
+    currentStake.shares &&
+    currentStake.inactiveShares &&
+    (currentStake.shares as BigInt).isZero() && 
+    (currentStake.inactiveShares as BigInt).isZero()
```

Comment:
Should this check to see if the active shares is zero?

For ex if an EOA has shares = 100, inactiveShares = 100 and activeShares = 0 we should remove the stakes from the stake list.

Reply:
@Robert-H-Leonard I think `currentStake.shares` are the active shares actually, there's no `activeShares` attribute.

Reply:
oh I understand what you're saying. `inactiveShares` means that there's pending stake for it to be withdrawn, so shouldn't it continue to be in the list?

Reply:
Yea I think we only care about if shares is zero. If shares is 0 and inactiveShares in non-zero that staker won't qualify for rewards. I think the logic you have works. It just means the staker won't be removed until the stake is fully withdrawn.